tests/e2e/testsLibrary/LanguageServerTests.ts

```diff
@@ -89,4 +95,22 @@ export class LanguageServerTests {
             }
         });
     }
+
```

Comment:
the logic calls activateBreakpoint.
I've added that method in case we want to do cleanup (for example move the debug section to a different part of the test), so that we could clean it up as well.

Should I remove the method if it's not used?

Reply:
I don't see the value of adding methods that are not used. Do you have some plans about using them in near future?

Reply:
Fair, can I keep the deactivateBreakpoint one though? we have no way to disable breakpoints once they are added if it's removed.

Reply:
ok, cleaned up